### PR TITLE
pcm_converter: Properly handle CONFIG_FORMAT_CONVERT_HIFI3

### PR DESCRIPTION
**Pull request by ktrzcinx**
Include config.h in header where this define is used

Signed-off-by: Karol Trzcinski <karolx.trzcinski@linux.intel.com>

### CODE REVIEW

**Comment by lgirdwood** [comment]
CI usual suspect.

**Comment by jajanusz** [comment]
> CI usual suspect. Need to wait for internal CI before merge it's not showing as pending...

CI team said that it's just how QB's GitHub plugin - it shows as pending only when it starts actually processing PR instead of notifying GH about all jobs in queue ...

File: src/include/sof/audio/pcm_converter.h [hunk 1]
```diff
@@ -14,6 +14,7 @@
 #ifndef __SOF_AUDIO_PCM_CONVERTER_H__
 #define __SOF_AUDIO_PCM_CONVERTER_H__
 
+#include <config.h>
```
Comment (jajanusz) [inline src/include/sof/audio/pcm_converter.h]: Includes should be sorted by increasing visibility, config is more global than ipc/* stuff, so it should be after it. 
config.h can only be included earlier if you have to decide what-to-include depending on CONFIG_ symbols

**Comment by lgirdwood** [comment]
> > CI usual suspect. Need to wait for internal CI before merge it's not showing as pending...
> 
> CI team said that it's just how QB's GitHub plugin - it shows as pending only when it starts actually processing PR instead of notifying GH about all jobs in queue ...

Hmm, seems like a race condition here with QB. We could potentially have a green circle before QB runs (which could result in a test failure).

**Comment by jajanusz** [comment]
> > > CI usual suspect. Need to wait for internal CI before merge it's not showing as pending...
> > 
> > 
> > CI team said that it's just how QB's GitHub plugin - it shows as pending only when it starts actually processing PR instead of notifying GH about all jobs in queue ...
> 
> Hmm, seems like a race condition here with QB. We could potentially have a green circle before QB runs (which could result in a test failure).

I guess, you can mark specific CIs as 'required' if you want them to be always on list, but idk if in that case it will prevent other maintainers to merge if that CIs fails (even if it's false negative).

**Comment by lgirdwood** [comment]
@zrombel fyi, scan is not running again ? Other CI issue is usual suspect.

**Comment by zrombel** [comment]
@lgirdwood KW servers performs better since new year but still unstable, we are monitoring and rerunning scan when necessary.

**Pull request merged**